docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  asvz-bot:
+    container_name: asvz-bot
+    image: asvz-bot
+    build:
+      context: .
+    environment:
```

Comment:
After testing this, I observed that the `environment` variables have precedence over the `env-file`. In case, we are following the `env-file` approach, we'd have to remove the `environment` section completly.

Reply:
I usually prefer to use multiple `env_file` configurations and will most likely use them in this project as well, but I'd like to leave this up to the user. For beginner docker users it is most likely simpler to manage a full copy of the docker-compose file for each configuration, rather than also working with multiple `.env` files, which is why I documented it this way.

Reply:
This led to issues with optional parameters such as level, so I had to add some additional checks in the python script.